components/centraldashboard/public/components/main-page.pug

```diff
@@ -59,8 +59,8 @@ app-drawer-layout.flex(narrow='{{narrowMode}}',
                     selected='{{namespace}}', hides, hidden$='[[hideNamespaces]]'
                     all-namespaces='[[allNamespaces]]')
                 footer#User-Badge
-                    iron-image.image(sizing='contain', fade,
-                        src='/assets/anon-user.png', title='[[user]]')
+                    a(target="_top", href="/logout")
+                        img(src='/assets/signout.png')
```

Comment:
Have you considered using an icon like `iron-icon(icon="exit-to-app")`that fits more with the overall design theme? 
You could find more suitable icons [here](http://www.twelvetone.tv/iron-icons-search/index.html)

Reply:
Also, the icon needs to have a title attribute suggesting `Logout` on hover

Reply:
@SachinVarghese , @StefanoFioravanzo please suggest any specific icon that works best? @SachinVarghese I dont see any thing specific to signout or logout in the link you shared above.

Reply:
@lalithvaka Yes I agree, none of them really matches the criteria, but maybe the closest one is to use `exit-to-app` as follows for now. 
```
iron-icon(icon="exit-to-app" title="logout")
```

Reply:
Per @avdaredevil  suggestion, have imported a new logout.svg image and used the iron-icon as well.